app/models/answer.rb

```diff
@@ -0,0 +1,41 @@
+class Answer < ActiveRecord::Base
+  has_many :comments, as: :commentable
```

Comment:
You could implement a "Voting" module that carries
```
has_voted(user)
vote_count
voted
```
and mix it into both Question and Answer